src/sections/Community/Handbook/repo-data.js

```diff
@@ -147,7 +152,7 @@ export const repo_data = [
         language: "React.js",
         maintainers_name: ["Antonette Caldwell"],
         link: ["https://layer5.io/community/members/antonette-caldwell"],
-        repository: "https://github.com/layer5io/sistent"
+        repository: "https://github.com/layer5io/sistent",
```

Comment:
Please add @aabidsofi19 as a maintainer here (and in the maintainers.md in that repo).

Reply:
Please add @Rexford74, too.